spec/rspec/mocks_spec.rb

```diff
@@ -132,7 +132,7 @@
         expect(defined?(ValueY)).to be_falsey
       end
 
-      it 'does not allow the stubbed constants to be used after the scope in before(:all)' do
+      it 'does not allow the stubbed constants to be used after the scope in before(:all)', :pending => RSpec::Support::Ruby.jruby_9000? do
```

Comment:
This is broken on JRuby 9000, but because of an internal `sprintf` which I'm not sure we can do anything about?